app/models/podcast.rb

```diff
@@ -14,8 +14,18 @@ class Podcast < BaseModel
 
   acts_as_paranoid
 
+  before_validation :set_defaults
+
   scope :published, -> { where('published_at IS NOT NULL AND published_at <= now()') }
 
+  def set_defaults
+    self.enclosure_template ||= enclosure_template_default
+  end
+
+  def enclosure_template_default
+    "https://#{ENV['DOVETAIL_HOST']}/{slug}/{guid}/{original_filename}"
```

Comment:
The template defines the overall order and shape of the url - does it have dovetail in there, is the original file name used, is the guid? It is optional - as you mention, we have that enclosure default that will return an f.prxu.org address.

The prefix is also optional, and will add a redirect prefix to whatever comes out for the enclosure url after the template (if there is one) has been applied to the enclosure url. It is only able to effect adding a prefix.

There is some overlap here, as a user could add a prefix in their template (which is what we have done in the past), but by adding this prefix, we have a way to create a user editable field that can only cause so much damage, where editing the template would be a bear.